src/components/Header/index.js

```diff
@@ -1,173 +1,97 @@
-                {isLoggedIn ? (
-                  <Profile user={user} />
-                ) : (
-                  <div data-testid="session-links" className="d-flex">
-                    {/* >= lg */}
-                    <a
-                      href={signupSSOUrl}
-                      onClick={trackOutboundLink}
-                      className="btn btn-lg btn-outline-dark d-none d-xl-block btn-session"
-                    >
-                      Sign up
-                    </a>
-                    {/* >= md */}
-                    <a
-                      href={loginSSOUrl}
-                      onClick={trackOutboundLink}
-                      className="btn btn-primary btn-lg d-none d-md-block btn-session"
-                    >
-                      Login
-                    </a>
-                    {/* small */}
-                    <a
-                      href={loginSSOUrl}
-                      onClick={trackOutboundLink}
-                      className="btn btn-primary btn-sm d-md-none d-xs-block d-sm-block btn-session"
-                    >
-                      Login
-                    </a>
-                  </div>
-                )}
-              </div>
-            </div>
-          </div>
+const Header = ({ user }: Props) => {
+  const isLoggedIn = has(user, 'id')
+  const renderLocalLinks = () => {
+    return (
+      <>
+        <div className="nav-item">
+          <Link to="/#faq" className="nav-link">
+            FAQ
+          </Link>
         </div>
-      </header>
-      <Collapse in={open}>
-        <div id="slider-nav" className="slider-nav d-xl-none">
-          <ul className="nav flex-column" role="navigation">
-            <li className="nav-item">
-              <Link to="/#faq" className="nav-link">
-                FAQ
+        {isLoggedIn && (
+          <>
+            <div className="nav-item" data-testid="actions-link">
+              <Link to="/app/actions" className="nav-link">
+                Actions
               </Link>
-            </li>
-            <li className="nav-item">
+            </div>
+            <div className="nav-item" data-testid="member-hub-link">
               <a
+                href="https://debtcollective.org/hub/"
                 className="nav-link"
-                href="https://membership.debtcollective.org"
                 target="_blank"
-                rel="noopener noreferrer"
-                onClick={trackOutboundLink}
               >
-                Donate
+                Member hub
               </a>
-            </li>
-          </ul>
-        </div>
-      </Collapse>
-    </>
-  )
-}
+            </div>
+          </>
+        )}
+      </>
+    )
+  }
 
-Header.propTypes = {
-  user: PropTypes.shape({
-    avatar_url: PropTypes.string,
-    username: PropTypes.string,
-    id: PropTypes.string
-  })
-}
+  const renderLinks = () => {
+    return (
+      <>
+        {renderLocalLinks()}
```

Comment:
I will try to stick to use JSX instead dropping methods like this one. Every time I see this it is often avoidable. JSX it is already a very good abstraction and declarative way to define a DOM structure, obscure the definition of this tends to be unnecessary.

If we really need to create an abstraction of a duplicated component why not to:

```
const LocalNavigationLinks = () => <div></div>

<div slot="header" className="header-links">
    <LocalNavigationLinks />
</div>
```

We can't do that?